hawc/apps/myuser/views.py

```diff
@@ -91,6 +91,10 @@ def get_context_data(self, **kwargs):
         )
         return context
 
+    def form_valid(self, form):
```

Comment:
Session is not easily accessible from a post_save signal, so I update the session variable here instead.